### PR TITLE
Adding vanity route + cert OCP template

### PR DESCRIPTION
**Pull request by oybed**
#### What is this PR About?
Follow-up PR to #403 to allow for OCP template use to create a vanity route with valid certificate

#### How do we test this?
Run the oc process command, e.g.:

```
oc process --local -f route-with-cert.yml -p NAME=test -p NAMESPACE=test -p FQDN=test.example.com -o yaml
```

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Review by garethahealy — DISMISSED** [review]

File: cert-manager-configs/.openshift/route-with-cert.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+kind: Template
+apiVersion: v1
```
[the remark this answers is not in the document]
Comment (oybed) [reply cert-manager-configs/.openshift/route-with-cert.yml]: Done - included the other templates/files in this directory as well.

File: cert-manager-configs/.openshift/route-with-cert.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - kind: Certificate
+    apiVersion: cert-manager.io/v1alpha2
+    metadata:
+      name: "${NAME}-certificate"
+      namespace: ${NAMESPACE}
+    spec:
+      secretName: "${NAME}-certificate"
+      issuerRef:
+        name: letsencrypt-production
+        kind: ClusterIssuer
```
Comment (pabrahamsson) [inline cert-manager-configs/.openshift/route-with-cert.yml:20]: Can we make these parameters, similar to the [Helm template](https://github.com/redhat-cop/containers-quickstarts/blob/master/cert-manager-configs/templates/Certificate.yaml#L12-L13)?

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Thank you @oybed this looks good. I left a request for additional parameters inline.

File: cert-manager-configs/.openshift/route-with-cert.yml (hunk 2 above)
Comment (oybed) [reply cert-manager-configs/.openshift/route-with-cert.yml]: @pabrahamsson done

**Review by pabrahamsson — APPROVED** [review]
/approve
/lgtm

**Comment by oybed** [comment]
@garethahealy mind approving the change so the bot can do its thing?

**Pull request merged**